src/rust/grapl-web-sec/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
Is this to live as a separate crate from the Actix web application that will be the frontend, or is this just a placeholder for now?

Reply:
separate crate